src/glb2-bounding-box.js

```diff
@@ -2,7 +2,7 @@ import gltf2BoundingBox from './gltf2-bounding-box';
 
 const glb2BoundingBox = {
 
-  computeBoundings(glb) {
+  computeBoundings(glb,precision=0) {
```

Comment:
Could you use a "parameters" object ? We may add more options in the future.

Something like

```js
  computeBoundings(glb, { precision = 0 } = {}) {
```